grunt/config/browserify.js

```diff
@@ -206,7 +206,7 @@ var addonsMin = {
 // The DOM Builds
 var dom = {
   entries: [
-    './build/node_modules/react-dom/lib/ReactDOMUMDEntry.js',
+    './build/node_modules/react-dom/lib/ReactDOMFiber.js',
```

Comment:
I think we should keep this line as it is but change what ReactDOMUMDEntry exports.